meta-balena-common/recipes-containers/resin-supervisor/resin-supervisor/update-resin-supervisor

```diff
@@ -96,56 +89,52 @@ else
     error_handler "no container engine detected"
 fi
 
-# Get target supervisor tag from API.
+# Get target supervisor details from API.
 # The script will exit if curl does not get a valid response.
 # Getting data separately before reading it fixes error handling.
-echo "Getting image name and tag..."
+echo "Getting image name and version..."
 if [ -n "$API_ENDPOINT" ] && [ -n "${UUID}" ] && [ -n "$_device_api_key" ] && data=$(curl --silent --header "Authorization: Bearer $_device_api_key" --header "User-Agent:" --compressed "${API_ENDPOINT}/v6/supervisor_release?\$select=supervisor_version,image_name&\$filter=should_manage__device/any(d:d/uuid%20eq%20'${UUID}')" | jq -e -r '.d[0].supervisor_version,.d[0].image_name'); then
```

Comment:
this query can also be updated to reference the new native release codepath

Reply:
@xginn8 `native release codepath` is a new term to me. Do I still use the `supervisor_release` endpoint that returns:
```
   {
      "id": 110122,
      "supervisor_version": "v10.3.7",
      "is_for__device_type": {
        "__id": 79,
        "__deferred": {
          "uri": "/resin/device_type(@id)?@id=79"
        }
      },
      "image_name": "registry2.balena-staging.com/v2/abe78e9c244253a5cdf79bf3225c5c76",
      "__metadata": {
        "uri": "/resin/supervisor_release(@id)?@id=110122"
      }
    },
```
Could you paste an example query of what you mean?

Reply:
thinking further about this, i'll sort it separately once we include this info in the new state endpoint.

Reply:
@xginn8 jusrt following up on this. Is the new state endpoint already including the release version?

Reply:
i think it makes the most sense to update this code independently once this PR is merged and we can access our supervisor release via the state endpoint: https://github.com/balena-io/open-balena-api/pull/504